scripts/lint_changelog_files.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
let's make sure the exit code is always right
```suggestion

set -euo pipefail
```

Reply:
BTW this led to some unexpected behaviour, as `grep` exits with `1` if there is no match (which is not an error in the case of both scripts). It took me a while to understand that...